compiler/quilt/nodes.py

```diff
@@ -83,16 +100,46 @@ def _group_keys(self):
         """
         return [name for name, child in self._items() if isinstance(child, GroupNode)]
 
-    def _keys(self):
-        """
-        keys directly accessible on this object via getattr or .
+    def _keys(self, recursive=False, iterator=False):
```

Comment:
Same here. Can you use the existing iterator instead of making this recursive?

Reply:
Sure, what iterator?  I didn't see one in either `nodes.py` or `package.py`.  Is there a helper somewhere else I should use?

Reply:
Since there is no iterator available, and `nodes.py` is going away, I've moved the code for the iterator to be contained inside the export() function.